### PR TITLE
Document Redpanda process utilization metric

### PR DESCRIPTION
**Pull request by shorgi**
Review deadline: Thu Mar 23

Fixes:
- #1413 

Preview:
- https://deploy-preview-1416--redpanda-documentation.netlify.app/docs/manage/monitoring/#cpu-usage

### CODE REVIEW

File: docs/manage/monitoring.mdx [hunk 1]
```diff
@@ -168,6 +168,12 @@ For the total CPU busy (non-idle) time, monitor [redpanda_cpu_busy_seconds_total
 rate(redpanda_cpu_busy_seconds_total[5m])
 ```
 
+#### Redpanda process utilization
+
+CPU utilization at the host-level can typically appear high (for example, 99-100% utilization) when I/O events like message arrival occur, while the actual Redpanda process utilization is low. This high host-level CPU utilization happens because Redpanda uses Seastar and poll-mode drivers (PMD) to bypass the kernel to interact with network packets directly, and by using PMD, Redpanda threads poll constantly as new packets arrive.
```
Comment (daisukebe) [inline docs/manage/monitoring.mdx:173]: @travisdowns , is this statement accurate and good to publish?

**Comment by Feediver1** [comment]
@travisdowns PR needs an engineering SME to review--can you take a look? Thx.

File: docs/manage/monitoring.mdx (hunk 1 above)
Comment (travisdowns) [reply docs/manage/monitoring.mdx:173]: We aren't really using PMD nor interacting with the network packets directly.

File: docs/manage/monitoring.mdx [hunk 2]
```diff
@@ -168,6 +168,12 @@ For the total CPU busy (non-idle) time, monitor [redpanda_cpu_busy_seconds_total
...
+
+For a measure of actual Redpanda process utilization, monitor the internal metric [`vectorized_reactor_utilization`](../../reference/internal-metrics-reference#vectorized_reactor_utilization).
```
Comment (travisdowns) [inline docs/manage/monitoring.mdx:175]: This seems odd in the sense that `redpanda_cpu_busy_seconds_total` and `vectorized_reactor_utilization` measure the same thing (presented differently: the latter is a gauge out of 100%, the former is a cumulative counter but fundamentally they are reporting the same thing).

**Review by Feediver1 — APPROVED** [review]
LGTM

File: docs/manage/monitoring.mdx (hunk 1 above)
Comment (shorgi) [reply docs/manage/monitoring.mdx]: @travisdowns - I've edited this statement, please review before the end of this week, May 26

File: docs/manage/monitoring.mdx (hunk 2 above)
Comment (shorgi) [reply docs/manage/monitoring.mdx]: @travisdowns - I'll be merging this updated PR by EOD Fri Jun 9, please comment on any changes before then, thanks

**Pull request merged**